src/Runner.Worker/ExecutionContext.cs

```diff
@@ -61,14 +61,15 @@ public interface IExecutionContext : IRunnerService
 
         bool EchoOnActionCommand { get; set; }
 
-        bool InsideComposite { get; }
+        bool IsEmbedded { get; }
```

Comment:
Renamed this since ExecutionContext lower level and ideally doesn't know about "Composite Actions"